tests/test_contextual_analysis.py

```diff
@@ -37,6 +40,8 @@ async def coro():
         self.assertEqual(refs.all_variables, variables)
         self.assertEqual(refs.local_variables, assigns)
         self.assertEqual(refs.undefined_variables, undefined)
+        if filters is not None:
```

Comment:
Same as previous comment. Assert that `refs.filters` is empty if not given.

```python
else:
    self.assertEqual(refs.filters, {})
```

Reply:
Done